README.md

```diff
@@ -1,70 +1,221 @@
+
+
+```cd my-folder```
+
+
+```git clone https://github.com/danielsafari143/math.git```
+
+
+### Install
+
+Install this project with:
+
+`cd my-project`
+
+
+`npm install`
+
+
+### Usage
+
+To run the project, execute the following command:
+
+`live server`
+
+`npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+`npm run test`
+
+`npm install --save-dev hint@7.x`
+
+`npx stylelint "**/*.{css,scss}"`
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+
+`github pages`
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Safari Hamuli**
+
+- GitHub: [@githubhandle](https://github.com/danielsafari143)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/safari-daniel-a01744251/)
+- GitHub: [@githubhandle](https://github.com/danielsafari143)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/safari-daniel-a01744251/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+- [ ] **[Set up react app]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+Most of us are pretty good at judging whether a project is relevant for us. We might not be the master of it but we might recognize that we’ve done something similar before and feel confident that we can pick it up and help.If you can’t find anything you think you are good at, consider looking at different projects that require less technical skill.If relevant for the project, you can also prepare some samples specific to the project to impress the client.The trick is to prepare some that are slightly different or watermarked, so they can’t use them in the project but close enough that they can see the potential If you like this project, kindly give it a star..
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank my colleague and Microverse for the opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- LICENSE -->
 
-### `npm run build` fails to minify
+## 📝 License <a name="license"></a>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ]  Kindly add the correct link to your MIT License when I click on the link in your license section it displays a 404 error page, you can add a license  [here](https://github.com/danielsafari143/math/community/license/new?branch=develop)